pkg/addon/addon.go

```diff
@@ -1393,7 +1393,7 @@ func (h *Installer) getAddonVersionMeetSystemRequirement(addonName string) strin
 			return ""
 		}
 		for _, version := range versions {
-			req := LoadSystemRequirements(version.Annotations["system"])
+			req := LoadSystemRequirements(version.Annotations)
```

Comment:
Why change the spec?

Reply:
Addon maybe only have one type of system requirement(vela or kubernetes),should not store them in one annotation.

Reply:
Are there backward compatibility issues?

Reply:
No, the CI of catalog have not add this mechanism. Refer to: https://github.com/kubevela/catalog/pull/397